packages/kotti-ui/source/kotti-style/forms/_input.scss

```diff
@@ -104,7 +92,7 @@ textarea.form-input {
 			border-bottom-right-radius: 0;
 		}
 
-		&:not(:first-child):not(:last-child) {
+		&:not(:first-child :last-child) {
```

Comment:
shouldn't this be comma separated? that doesn't look right

Reply:
```suggestion
		&:not(:first-child, :last-child) {
```